server-spi-private/src/main/java/org/keycloak/models/utils/RepresentationToModel.java

```diff
@@ -1785,6 +1780,15 @@ public static UserModel createUser(KeycloakSession session, RealmModel newRealm,
         return user;
     }
 
+    public static void createFederatedIdentities(UserRepresentation userRep, KeycloakSession session, RealmModel realm, UserModel user) {
```

Comment:
Can you revert the change to extract this into a separate method? We don't need a public method like this when it's only used within a single place in the same class